Documentation/Storage-Configuration/Ceph-CSI/ceph-csi-drivers.md

```diff
@@ -238,3 +238,40 @@ parameters:
 ```
 
 * PVCs created using the new storageclass will be encrypted.
+
+
+### Enable Read affinity for RBD volumes
+
+Ceph CSI supports mapping RBD volumes with krbd options
+`"read_from_replica=localize,crush_location=type1:value1|type2:value2"` to
+allow serving reads from the most local OSD (according to OSD locations as
+defined in the CRUSH map).
+Refer [krbd-options](https://docs.ceph.com/en/latest/man/8/rbd/#kernel-rbd-krbd-options)
+for more details.
+
+Execute the following steps:
+
+* Add labels to Kubernetes nodes following procedure similar to how it is mentioned in
+[osd-topology](../../CRDs/Cluster/ceph-cluster-crd.md#osd-topology).
+
+* Update the `rook-ceph-operator-config` configmap and patch it using the following
+command.
+```console
+kubectl patch cm rook-ceph-operator-config -nrook-ceph -p $'data:\n "CSI_ENABLE_READ_AFFINITY": "true"'
```

Comment:
How about a patch command that just replaces the default `false` value? Otherwise this command seems like it would add a duplicate setting if that setting is already in the configmap.

Reply:
> How about a patch command that just replaces the default `false` value? Otherwise this command seems like it would add a duplicate setting if that setting is already in the configmap.

that command would fail in case the existing configmap does not have the default being set to false. 
But since we'll have it turned on by default, I'll move this command to the bottom and 
give them instructions to turn it off if required (patch in this case too).